.bingo/Variables.mk

```diff
@@ -22,6 +22,12 @@ $(BINGO): .bingo/bingo.mod
 	@echo "(re)installing $(GOBIN)/bingo-v0.2.3"
 	@cd .bingo && $(GO) build -mod=mod -modfile=bingo.mod -o=$(GOBIN)/bingo-v0.2.3 "github.com/bwplotka/bingo"
 
+GO_JUNIT_REPORT := $(GOBIN)/go-junit-report-v0.9.1
```

Comment:
Shouldn't you use `$(GO_JUNIT_REPORT)` instead of `$(GOBIN)/go-junit-report-v0.9.1`